vine/src/compile/build_stages/control_flow.rs

```diff
@@ -95,9 +95,11 @@ impl Compiler<'_> {
     Port::Erase
   }
 
-  pub(super) fn lower_break(&mut self) -> Port {
-    let (fork, _stage) = self.loop_target.unwrap();
-    self.diverge(fork);
+  pub(super) fn lower_break(&mut self, r: &Expr) -> Port {
+    let r = self.lower_expr_value(r);
+    let ret = self.return_target.unwrap();
```

Comment:
this is causing the `fib_repl` failure; `break` is now identical to `return`

Reply:
yes break_target now needs a local

Reply:
but why would fib_repl fail if lower_break never gets called (my assumption given there's no break command in it)

Reply:
`fib_repl` calls `std::io::read_line`, which uses `break`

Reply:
(i swear I added a println to see)

Reply:
(by default the compiled code gets sent to stdout so you might have more luck with `eprintln!`)